tasks/main.yml

```diff
@@ -21,7 +21,6 @@
     owner: root
     group: root
     mode: 0644
-  when: apt_etc_apt_apt_conf | length > 0
```

Comment:
This is not backwards compatible

Reply:
That's right. Does this configuration file exist on any default installation? At least not on my Ubuntu 18.04/20.04 systems, so I wouldn't mind creating an empty file.

Reply:
If that's the case then I see no problem